main.js

```diff
@@ -46,18 +46,26 @@ function changeGarment() {
 }
 
 function selectButton() {
-  if(event.target.classList.contains('item-button')) {
-    var garmentButtons = event.target.parentElement.querySelectorAll('.item-button')
-    for(var i = 0; i < garmentButtons.length; i++) {
-      garmentButtons[i].classList.remove('selected-button')
+  if (event.target.classList.contains('item-button')) {
+    if (event.target.classList.contains('selected-button')) {
+      event.target.classList.remove('selected-button');
+    } else {
+      var garmentButtons = event.target.parentElement.querySelectorAll('.item-button')
+      for(var i = 0; i < garmentButtons.length; i++) {
+        garmentButtons[i].classList.remove('selected-button')
+      }
+      event.target.classList.add('selected-button');
     }
-    event.target.classList.add('selected-button');
   }
 }
 
 function showGarmentOnBear(garmentName, garmentBox) {
   var garmentContainer = document.getElementById(garmentBox);
-  garmentContainer.classList.add(garmentName);
+  if (garmentContainer.classList.contains(garmentName)) {
+    garmentContainer.classList.remove(garmentName);
+  } else {
+    garmentContainer.classList.add(garmentName);
```

Comment:
We need to add a reset in here, it looks like we are running into an issue where we keep adding classes to the container and run into the top-down hierarchy issue with classes. 

```
function showGarmentOnBear(garmentName, garmentBox) {
  var garmentContainer = document.getElementById(garmentBox);
  if (garmentContainer.classList.contains(garmentName)) {
    garmentContainer.classList.remove(garmentName);
  } else {
    var className = garmentContainer.className.split(" ")[0];
    garmentContainer.className = className;
    garmentContainer.classList.add(garmentName);
  }
```
}